### ISSUE TITLE
Connect to the camera with CameraSK-Cpp via wifi

### ISSUE THREAD

**Issue by kin0201** [Insta360Develop/CameraSDK-Cpp#7]
Can I connect the camera to my linux muachine with CameraSK-Cpp via wifi?

**Comment by CodeSpoof** [comment]
No, wifi is only usable in the mobile SDK, OSC and ProCamera API.
Source: [https://www.insta360.com/de/sdk/home](https://www.insta360.com/de/sdk/home)

**Comment by Tianweihaihaihai** [comment]
wifi is not available. and camera api is only used on prefessional camera like pro2
